readthedocs.yml

```diff
@@ -1,6 +1,14 @@
+version: 2
+
+build:
+  image: latest
+
```

Comment:
Let's add: "formats: all" here (as per [this](https://docs.readthedocs.io/en/latest/config-file/v2.html#formats)) since the default in v1 was "all" while in v2 is nothing.

Reply:
Actually not sure if this is even necessary....thoughts? @davidwagner

Reply:
Not sure either, but we can add it for explicitness.